candle-datasets/Cargo.toml

```diff
@@ -18,3 +18,6 @@ intel-mkl-src = { workspace = true, optional = true }
 memmap2 = { workspace = true }
 tokenizers = { workspace = true, features = ["onig"] }
 rand = { workspace = true }
+thiserror = { workspace = true }
+parquet = { version = "45.0.0"}
```

Comment:
Why not define the parquet version in the top level `Cargo.toml`? I would think it's better to centralize everything here even if there is a single crate using it for now.
Also do you have a sense of how big of a dependency this crate ends up being?

Reply:
`Cargo bloat --release --crates`:

```
6.6%  15.3%   2.2MiB parquet
 6.1%  14.1%   2.1MiB arrow_cast
 ```
 
 ```
 6.6%  15.3%   2.2MiB parquet
 6.1%  14.1%   2.1MiB arrow_cast
 5.2%  12.0%   1.7MiB candle_core
 3.1%   7.2%   1.1MiB std
 1.9%   4.4% 653.7KiB arrow_select
 1.9%   4.3% 647.1KiB arrow_array
 1.8%   4.2% 628.5KiB [Unknown]
 1.6%   3.8% 561.3KiB image
 1.4%   3.3% 491.6KiB rustls
 0.9%   2.1% 311.3KiB zstd_sys
 0.7%   1.6% 244.2KiB ureq
 0.6%   1.5% 222.2KiB arrow_ipc
 0.6%   1.5% 219.7KiB ring
 0.6%   1.4% 208.3KiB brotli_decompressor
 0.6%   1.4% 204.9KiB brotli
 0.5%   1.1% 160.6KiB arrow_data
 0.4%   0.9% 141.5KiB openssl
 0.4%   0.8% 126.9KiB num_bigint
 0.3%   0.8% 114.0KiB indicatif
 0.3%   0.7% 111.8KiB lz4_sys
 6.6%  15.4%   2.2MiB And 95 more crates. Use -n N to show more.
43.1% 100.0%  14.6MiB .text section size, the file size is 33.9MiB
```

So relatively sizeable.